scss/civicrm/contact/_tabs.scss

```diff
@@ -11,11 +13,11 @@
       border-radius: 0;
       box-shadow: $box-shadow;
       float: left;
-      padding: 0 0 99999px;
       margin-bottom: -99999px;
+      padding: 0 0 99999px;
       width: 200px;
 
-      &~.ui-tabs-panel {
+      ~.ui-tabs-panel {
```

Comment:
so was this selector wrong all along?

Reply:
Yeah, we actually don't need & sign for such selection, & is generally needed when we want to combine multiple classes,
like
```scss
.class1 {
  &.class2 {
    //style here
  }
}
```

The above one with have same output weather we use & or not. (Though, this is stylelint change nothing to do with current ticket)